cypress/e2e/shared/explorer.test.ts

```diff
@@ -66,6 +66,7 @@ describe('DataExplorer', () => {
       cy.get('.cf-tree-nav--toggle').click()
       // Can't use the testID to select this nav item because Clockface is silly and uses the same testID twice
       // Issue: https://github.com/influxdata/clockface/issues/539
+      cy.getByTestID('nav-item-load-data').click()
       cy.get('.cf-tree-nav--sub-item-label')
```

Comment:
this comment is supposed to be on the line underneath this change

Reply:
So true, i'll fix that 👍